client/src/components/Score.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Score({score}) {
```

Comment:
Another good use of props to have a simple, well encapsulated component which receives state from a parent component.